pkgs/top-level/python-packages.nix

```diff
@@ -9367,6 +9367,29 @@ in {
     };
   };
 
+  subliminal = buildPythonApplication rec {
```

Comment:
I read that I should add it here in the nixpkgs manual, I'll move it to its own dir and then import.